lib/linux_admin/disk.rb

```diff
@@ -8,16 +8,23 @@ class Disk
 
     attr_accessor :path, :model
 
+    # Collect local disk information via the lsblk command. Only disks with a
```

Comment:
I don't think we should mention the implelementation of how we get the information in the docs.  LinuxAdmin is meant to be an abstraction, and we may choose a different tech for different stacks.